data/set/lattice.lean

```diff
@@ -219,6 +227,14 @@ theorem bUnion_pair (a b : α) (s : α → set β) :
   (⋃ x ∈ ({a, b} : set α), s x) = s a ∪ s b :=
 by rw insert_of_has_insert; simp [union_comm]
 
+@[simp] -- complete_boolean_algebra
+theorem compl_bUnion (s : set α) (t : α → set β) : - (⋃ i ∈ s, t i) = (⋂ i ∈ s, - t i) :=
+ext (λ x, by simp)
+
+-- classical -- complete_boolean_algebra
```

Comment:
why has `compl_bUnion` a `[simp]` but `compl_bInter` doesn't?

Reply:
I copied these annotations and comments from the `compl_Union`/`compl_Inter` versions. I assume it is because `compl_Union` holds constructively while `compl_Inter` does not.